Content.Server/Administration/Commands/RejuvenateCommand.cs

```diff
@@ -90,9 +90,9 @@ public static void PerformRejuvenate(EntityUid target)
                 entMan.RemoveComponent<JitteringComponent>(target);
             }
 
-            if (entMan.TryGetComponent<DiseaseCarrierComponent>(target, out var carrier))
+            if (entMan.HasComponent<DiseaseCarrierComponent>(target))
             {
-                EntitySystem.Get<DiseaseSystem>().CureAllDiseases(carrier);
+                EntitySystem.Get<DiseaseSystem>().CureAllDiseases(target);
```

Comment:
Honestly, just keep the `TryGetComponent` and pass the result to the method. That's what resolves were intended for.